src/exampleTypes/ResultTable/ResultTableFooter.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import F from 'futil'
+import _ from 'lodash/fp'
+import { Grid, GridItem, PagerItem } from '../../greyVest'
+import ResultPager from '../ResultPager'
+import { contexturifyWithoutLoader } from '../../utils/hoc'
+
+let getTotalRecords = F.cascade([
```

Comment:
Use the util?

Reply:
the util is for `results`, not `totalRecords` - though I could've exported them both from utils (probably from a generic `getResultsContext` that may or may not also be exported)

will try to keep this in mind for later

Reply:
Oh :P